charts/rstudio-pm/values.yaml

```diff
@@ -85,6 +85,8 @@ service:
   nodePort: false
   # -- The Service port. This is the port your service will run under.
   port: 80
+  # -- The port to forward to on the Package Manager pod. Also see pod.port
```

Comment:
There actually isn't a `pod.port` option in the values like in Connect. It looks like `4242` is hard coded here: https://github.com/rstudio/helm/blob/9ee39769c91cea59a705eac4e1578ecab727b26f/charts/rstudio-pm/templates/deployment.yaml#L109

Might make sense to add this value as well.